src/common/pick_address.h

```diff
@@ -79,4 +79,8 @@ int get_iface_numa_node(
   const std::string& iface,
   int *node);
 
+int get_bond_numa_node(
```

Comment:
can/should this be combined with the get_iface_numa_node so the caller doesn't have to explicitly check both functions?